apps/site/src/lib/components/Navbar/Navbar.tsx

```diff
@@ -85,12 +87,15 @@ function Navbar({ identity }: NavbarProps) {
 							isLightVersion
 						/>
 					) : (
-						<Button
-							text="Logout"
+						<a
 							href="/logout"
-							usePrefetch={false}
-							isLightVersion
-						/>
+							className={clsx(
+								buttonStyles.button,
+								buttonStyles.lightButton,
+							)}
+						>
+							Logout
```

Comment:
Technically out of scope, but "Log In" and "Log Out" as verbs rather than adjectives would make more sense as action descriptions.

Reply:
It's a minor change so I'll just add it in this PR.